### PR TITLE
Add Amazon SES Logo

### LINKED ISSUE #8660: Amazon SES
### Brand Name

Amazon SES

### Website

https://aws.amazon.com/ses/

### Popularity Metric

The SimilarWeb rank is 454,126

### Official Resources for Icon and Color

aws-ses-logo

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by RiteshK-611**
**Issue:** closes #8660 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
This commit will resolve issue #8660 by adding Amazon SES (Simple Email Service) logo in amazonses.svg file and by updating simple-icons.json file

### Preview
amazonsimpleemailservice

### CODE REVIEW

File: icons/amazonses.svg
Comment (adamrusted) [inline icons/amazonses.svg:1]: Spotting [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/f9e3ce28-adaa-4c7d-bc30-0b48a82d828d) between your PR in black, and the official icon in red. Could you take a look at this please @RiteshK-611?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -649,6 +649,20 @@
...
+            "hex": "DD344C",
+            "source": "https://aws.amazon.com/architecture/icons/",
+            "guidelines": "https://aws.amazon.com/architecture/icons/",
+            "aliases": {
+                "aka": [
+                    "Amazon Simple Email Service",
+                    "Amazon SES",
+                    "AWS Simple Email Service",
+                    "AWS SES"
+                ]
```
Comment (adamrusted) [inline _data/simple-icons.json:663]: I'm not sure we need all/any of these aliases. Perhaps have AWS SES as the abbreviation, and Amazon Simple Email Service as the full title?

**Review by adamrusted — CHANGES_REQUESTED** [review]
Couple changes to be made to these please @RiteshK-611!

File: _data/simple-icons.json [hunk 2]
```diff
@@ -649,6 +649,18 @@
...
+            "title": "Amazon SES",
+            "hex": "DD344C",
+            "source": "https://aws.amazon.com/architecture/icons/",
+            "guidelines": "https://aws.amazon.com/architecture/icons/",
+            "aliases": {
+                "aka": [
+                    "Amazon Simple Email Service",
+                    "AWS SES"
+                ]
+            }
```
Comment (adamrusted) [inline _data/simple-icons.json:662]:
```suggestion
            "title": "AWS SES",
            "hex": "DD344C",
            "source": "https://aws.amazon.com/architecture/icons/",
            "guidelines": "https://aws.amazon.com/architecture/icons/",
            "aliases": {
                "aka": [
                    "Amazon Simple Email Service"
                ]
            }
```

File: icons/amazonses.svg
Comment (adamrusted) [inline icons/amazonses.svg:1]: It looks like you're using a precision of 2 decimals here, and that's resulting in [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/cea4bb2a-5753-4026-b544-482133f58802) between the source and your final SVG. Could you try with 3 or 4 decimals of precision with SVGO?

Comment (RiteshK-611) [reply icons/amazonses.svg:1]: Done!

File: icons/amazonsimpleemailservice.svg
Comment (adamrusted) [inline icons/amazonsimpleemailservice.svg:1]: Your PR seems to be in 2 decimals of precision, leading to some [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/5e7642dd-a70a-4c52-b7ed-c073a82de506) like this. Please regenerate the icon from source using a precision of 3 decimals - that should fix this.

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/amazonsimpleemailservice.svg
Comment (RiteshK-611) [reply icons/amazonsimpleemailservice.svg:1]: Check!

Comment (adamrusted) [inline icons/amazonsimpleemailservice.svg:1]: Okay... 4 might be necessary...
image

Comment (adamrusted) [reply icons/amazonsimpleemailservice.svg:1]: Which version of the icon are you using @RiteshK-611? I'm comparing all of these against the 64*64 SVG icons - and the differences may just be down to the size of the source icon!

**Review by adamrusted — COMMENTED** [review]
I'm still seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/0ce482da-8e41-463c-bab6-454da843b31d) but at this point I can't tell if it's me! @LitoMore - could you lend a second pair of eyes on this one?

File: icons/amazonsimpleemailservice.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...57 3.8571 21c0-1.0286.7715-1.8857 1.7143-2.1v-1.7571c0-.2572.1715-.4286.4286-.4286h5.5714v-2.5714H6c-.2571 0-.4286-.1715-.4286-.4286V6c0-.2571.1715-.4286.4286-.4286h12c.2571 0 .4286.1715.4286.4286v7.7143c0 .2571-.1715.4286-.4286.4286h-5.5714v2.5714H18c.2571 0 .4286.1714.4286.4286V18.9c.9857.2143 1.7143 1.0714 1.7143 2.1zM12 20.5714c-.6857 0-1.2857.6-1.2857 1.2857 0 .6858.6 1.2858 1.2857 1.2858.6857 0 1.2857-.6 1.2857-1.2858 0-.6857-.6-1.2857-1.2857-1.2857zm-6-.8571c-.6857 0-1.2857.6-1.2857 1.2857 0 .6857.6 1.2857 1.2857 1.2857.6857 0 1.2857-.6 1.2857-1.2857 0-.6857-.6-1.2857-1.2857-1.2857zM24 12c0 2.5286-.7714 4.9714-2.2714 6.9857l-.6857-.4714c1.3714-1.9286 2.1-4.1572 2.1-6.5143C23.1429 5.8714 18.1286.8571 12 .8571 5.8714.8571.8571 5.8714.8571 12c0 2.3571.7286 4.5857 2.1 6.5143l-.6857.4714C.7714 16.9714 0 14.5286 0 12 0 5.4 5.4 0 12 0s12 5.4 12 12ZM6.4286 6.9429v5.8285l3.3-3zm10.4143-.5143H7.157L12 10.5857Zm.7285 6.3428V6.943l-3.3 2.8285zm1.7143 8.2715c0-.7286-.6-1.2858-1.2857-1.2858-.6857 0-1.2857.5572-1.2857 1.2858 0 .6857.6 1.2857 1.2857 1.2857.6857 0 1.2857-.6 1.2857-1.2857zM7.1143 13.3286h9.7714l-3.3-2.9572L12.3 11.4857q-.1286.1286-.3.1286t-.2571-.1286l-1.3286-1.1143z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/amazonsimpleemailservice.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Amazon Simple Email Service</title><path d="M11.9996 0C5.3833 0 0 5.3834 0 11.9998c0 2.5316.7813 4.9544 2.2599 7.0051l.6955-.5014C1.5827 16.5993.8571 14.3505.8571 11.9998.8571 5.856 5.856.8572 12.0004.8572c6.144 0 11.1425 4.999 11.1425 11.1426 0 2.3508-.7256 4.5995-2.0983 6.5037l.6955.5014C23.2187 16.9542 24 14.5314 24 11.9998 24 5.3834 18.6163 0 11.9996 0z"/><path d="M6 16.7142a.4285.4285 0 0 0-.4286.4285v1.7598c-.9643.2048-1.7143 1.0822-1.7143 2.0974 0 1.1615.9815 2.143 2.1429 2.143s2.1429-.9815 2.1429-2.143c0-1.0152-.75-1.8926-1.7143-2.0974v-1.3312h5.1428v2.1883c-.9643.2049-1.7143 1.0822-1.7143 2.0975C9.8571 23.0186 10.8386 24 12 24s2.1429-.9814 2.1429-2.1429c0-1.0153-.75-1.8926-1.7143-2.0975v-2.1883h5.1428v1.3312c-.9643.2048-1.7143 1.0822-1.7143 2.0974 0 1.1615.9815 2.143 2.1429 2.143s2.1429-.9815 2.1429-2.143c0-1.0152-.75-1.8926-1.7143-2.0974v-1.7598A.4285.4285 0 0 0 18 16.7142h-5.5714v-2.5715H18c.237 0 .4286-.192.4286-.4286V5.9997A.4285.4285 0 0 0 18 5.571H6a.4285.4285 0 0 0-.4286.4286v7.7144c0 .2366.1916.4286.4286.4286h5.5714v2.5715H6zm1.2857 4.2857c0 .697-.5889 1.2858-1.2857 1.2858s-1.2857-.5889-1.2857-1.2858c0-.6968.5889-1.2857 1.2857-1.2857S7.2857 20.3031 7.2857 21zm12 0c0 .697-.5889 1.2858-1.2857 1.2858s-1.2857-.5889-1.2857-1.2858c0-.6968.5889-1.2857 1.2857-1.2857s1.2857.5889 1.2857 1.2857zm-1.7143-8.248L14.259 9.7703l3.3124-2.8389v5.8205zm-.7298-6.3236-4.842 4.1499-4.8412-4.15h9.6832zm-10.413.5031L9.741 9.7707 6.4286 12.752V6.9314zm.6878 6.3541 3.2807-2.9525 1.3239 1.135a.4253.4253 0 0 0 .2786.1032.4253.4253 0 0 0 .2785-.1033l1.3243-1.1349 3.2812 2.9525H7.1164zM12 20.5714c.6968 0 1.2857.5888 1.2857 1.2857 0 .6969-.5889 1.2857-1.2857 1.2857s-1.2857-.5888-1.2857-1.2857c0-.6969.5889-1.2857 1.2857-1.2857z"/></svg>
```
I am also seeing [differences in image](https://github.com/simple-icons/simple-icons/assets/35000807/8a8a3f7e-6d96-47c0-a372-819ea5eb34f2)


This should fix it.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

File: icons/amazonsimpleemailservice.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...1.3312h5.1428v2.1883c-.9643.2049-1.7143 1.0822-1.7143 2.0975C9.8571 23.0186 10.8386 24 12 24s2.1429-.9814 2.1429-2.1429c0-1.0153-.75-1.8926-1.7143-2.0975v-2.1883h5.1428v1.3312c-.9643.2048-1.7143 1.0822-1.7143 2.0974 0 1.1615.9815 2.143 2.1429 2.143s2.1429-.9815 2.1429-2.143c0-1.0152-.75-1.8926-1.7143-2.0974v-1.7598A.4285.4285 0 0 0 18 16.7142h-5.5714v-2.5715H18c.237 0 .4286-.192.4286-.4286V5.9997A.4285.4285 0 0 0 18 5.571H6a.4285.4285 0 0 0-.4286.4286v7.7144c0 .2366.1916.4286.4286.4286h5.5714v2.5715H6zm1.2857 4.2857c0 .697-.5889 1.2858-1.2857 1.2858s-1.2857-.5889-1.2857-1.2858c0-.6968.5889-1.2857 1.2857-1.2857S7.2857 20.3031 7.2857 21zm12 0c0 .697-.5889 1.2858-1.2857 1.2858s-1.2857-.5889-1.2857-1.2858c0-.6968.5889-1.2857 1.2857-1.2857s1.2857.5889 1.2857 1.2857zm-1.7143-8.248L14.259 9.7703l3.3124-2.8389v5.8205zm-.7298-6.3236-4.842 4.1499-4.8412-4.15h9.6832zm-10.413.5031L9.741 9.7707 6.4286 12.752V6.9314zm.6878 6.3541 3.2807-2.9525 1.3239 1.135a.4253.4253 0 0 0 .2786.1032.4253.4253 0 0 0 .2785-.1033l1.3243-1.1349 3.2812 2.9525H7.1164zM12 20.5714c.6968 0 1.2857.5888 1.2857 1.2857 0 .6969-.5889 1.2857-1.2857 1.2857s-1.2857-.5888-1.2857-1.2857c0-.6969.5889-1.2857 1.2857-1.2857z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/amazonsimpleemailservice.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Amazon Simple Email Service</title><path d="M11.9996 0C5.3833 0 0 5.3834 0 11.9998c0 2.5316.7813 4.9544 2.2599 7.0051l.6955-.5014C1.5827 16.5993.8571 14.3505.8571 11.9998.8571 5.856 5.856.8572 12.0004.8572c6.144 0 11.1425 4.999 11.1425 11.1426 0 2.3508-.7256 4.5995-2.0983 6.5037l.6955.5014C23.2187 16.9542 24 14.5314 24 11.9998 24 5.3834 18.6163 0 11.9996 0zM6 16.7142a.4285.4285 0 0 0-.4286.4285v1.7598c-.9643.2048-1.7143 1.0822-1.7143 2.0974 0 1.1615.9815 2.143 2.1429 2.143s2.1429-.9815 2.1429-2.143c0-1.0152-.75-1.8926-1.7143-2.0974v-1.3312h5.1428v2.1883c-.9643.2049-1.7143 1.0822-1.7143 2.0975C9.8571 23.0186 10.8386 24 12 24s2.1429-.9814 2.1429-2.1429c0-1.0153-.75-1.8926-1.7143-2.0975v-2.1883h5.1428v1.3312c-.9643.2048-1.7143 1.0822-1.7143 2.0974 0 1.1615.9815 2.143 2.1429 2.143s2.1429-.9815 2.1429-2.143c0-1.0152-.75-1.8926-1.7143-2.0974v-1.7598A.4285.4285 0 0 0 18 16.7142h-5.5714v-2.5715H18c.237 0 .4286-.192.4286-.4286V5.9997A.4285.4285 0 0 0 18 5.571H6a.4285.4285 0 0 0-.4286.4286v7.7144c0 .2366.1916.4286.4286.4286h5.5714v2.5715H6zm1.2857 4.2857c0 .697-.5889 1.2858-1.2857 1.2858s-1.2857-.5889-1.2857-1.2858c0-.6968.5889-1.2857 1.2857-1.2857S7.2857 20.3031 7.2857 21zm12 0c0 .697-.5889 1.2858-1.2857 1.2858s-1.2857-.5889-1.2857-1.2858c0-.6968.5889-1.2857 1.2857-1.2857s1.2857.5889 1.2857 1.2857zm-1.7143-8.248L14.259 9.7703l3.3124-2.8389v5.8205zm-.7298-6.3236-4.842 4.1499-4.8412-4.15h9.6832zm-10.413.5031L9.741 9.7707 6.4286 12.752V6.9314zm.6878 6.3541 3.2807-2.9525 1.3239 1.135a.4253.4253 0 0 0 .2786.1032.4253.4253 0 0 0 .2785-.1033l1.3243-1.1349 3.2812 2.9525H7.1164zM12 20.5714c.6968 0 1.2857.5888 1.2857 1.2857 0 .6969-.5889 1.2857-1.2857 1.2857s-1.2857-.5888-1.2857-1.2857c0-.6969.5889-1.2857 1.2857-1.2857z"/></svg>
```
The linter error should be fixed now

**Review by PratyushJayachandran — APPROVED** [review]

**Pull request merged**